src/hooks/xcm/useGetChains.tsx

```diff
@@ -0,0 +1,100 @@
+export type ChainTransferData = {
+    name: string,
+    id: string,
+    assets?: string[],
+    destWeight?: number
+}
+
+export type ChainType = {
+    name: string,
+    id: string,
+    supportedTransfers: ChainTransferData[]
+};
+
+
+export const CHAINS : ChainType[] = [
+    {
+        name: "Kusama",
+        id: "parent",
+        supportedTransfers: []
+    },
+    {
+        name: "Basilisk",
+        id: "2084",
+        supportedTransfers: [
+            { name : "Karura",
+                id: "2000",
+                assets: ["2"], // Asset 2 is registered as KUSD in dev chainspec
+                destWeight: 5e9
```

Comment:
Is this the existential deposit, or the call weight/price?

Reply:
it is basically maximum weight you are willing to pay for on destination chain. If it's incorrect, the transfer is stopped by a  `Barrier` error or `TooExpensive` error. 

but it is not really for user to choose. We set this for each supported chain to make sure it passes. 

When you do a transfer, the fee is taken from the transferred amount.